provisioning/roles/gulp/tasks/main.yml

```diff
@@ -3,7 +3,12 @@
   with_items:
     - { src: package.json, dest: "{{ gulp_package_json_path }}" }
     - { src: Gulpfile.js, dest: "{{ gulp_directory }}/Gulpfile.js" }
+
+- name: Create Gulp config file
+  template: src={{ item.src }} dest={{ item.dest }} force=no
+  with_items:
```

Comment:
Could you please remove the `with_items` and directly put the parameters in the `template` element?